.github/workflows/tests-8.x.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: ['8.0', '8.1']
+        version: ['8.0', '8.1', '8.2']

```

Comment:
Sorry about that, the 8.2 can't be there either, I thought I've already migrated the codebase to 8.3, in that case it should be there, but as it is, only 8.0 and 8.1 should be there. The version this is built against is tested in `tests.yaml`.

Reply:
No problem, fixed.

Thank you for clarification.